lib/open_api_spex/cast/discriminator.ex

```diff
@@ -98,4 +99,14 @@ defmodule OpenApiSpex.Cast.Discriminator do
   defp error(message, %{schema: %{discriminator: %{propertyName: property}}} = ctx) do
     Cast.error(ctx, {message, property})
   end
+
+  defp _locate_schemas(schemas, ctx_schemas) do
```

Comment:
Can you remove the leading underscore from this function? That's not a common practice in Elixir.